tests/cases/standalone/common/promql/label.result

```diff
@@ -242,6 +229,105 @@ TQL EVAL(0, 15, '5s') {__name__="test",host="host1"} + label_replace(vector(1),
+| 1970-01-01T00:00:00 | 2   | host2 | host2 | idc1       |
+| 1970-01-01T00:00:05 | 1   | host2 | host1 | idc1       |
+| 1970-01-01T00:00:05 | 2   | host2 | host2 | idc1       |
+| 1970-01-01T00:00:05 | 3   | host2 | host1 | idc2:zone1 |
+| 1970-01-01T00:00:05 | 4   | host2 | host2 | idc2       |
+| 1970-01-01T00:00:10 | 1   | host2 | host1 | idc1       |
+| 1970-01-01T00:00:10 | 2   | host2 | host2 | idc1       |
+| 1970-01-01T00:00:10 | 3   | host2 | host1 | idc2:zone1 |
+| 1970-01-01T00:00:10 | 4   | host2 | host2 | idc2       |
+| 1970-01-01T00:00:10 | 5   | host2 | host1 | idc3:zone2 |
+| 1970-01-01T00:00:10 | 6   | host2 | host2 | idc3       |
+| 1970-01-01T00:00:15 | 1   | host2 | host1 | idc1       |
+| 1970-01-01T00:00:15 | 2   | host2 | host2 | idc1       |
+| 1970-01-01T00:00:15 | 3   | host2 | host1 | idc2:zone1 |
+| 1970-01-01T00:00:15 | 4   | host2 | host2 | idc2       |
+| 1970-01-01T00:00:15 | 5   | host2 | host1 | idc3:zone2 |
+| 1970-01-01T00:00:15 | 6   | host2 | host2 | idc3       |
+| 1970-01-01T00:00:15 | 7   | host2 | host1 | idc4:zone3 |
+| 1970-01-01T00:00:15 | 8   | host2 | host2 | idc4       |
++---------------------+-----+-------+-------+------------+
+
+-- Empty regex with not existing source label, but replacement is empty--
+-- SQLNESS SORT_RESULT 3 1
+TQL EVAL(0, 15, '5s') label_replace(test, "host2", "", "instance", "");
+
++---------------------+-----+-------+------------+
+| ts                  | val | host  | idc        |
++---------------------+-----+-------+------------+
+| 1970-01-01T00:00:00 | 1   | host1 | idc1       |
+| 1970-01-01T00:00:00 | 2   | host2 | idc1       |
+| 1970-01-01T00:00:05 | 1   | host1 | idc1       |
+| 1970-01-01T00:00:05 | 2   | host2 | idc1       |
+| 1970-01-01T00:00:05 | 3   | host1 | idc2:zone1 |
+| 1970-01-01T00:00:05 | 4   | host2 | idc2       |
+| 1970-01-01T00:00:10 | 1   | host1 | idc1       |
+| 1970-01-01T00:00:10 | 2   | host2 | idc1       |
+| 1970-01-01T00:00:10 | 3   | host1 | idc2:zone1 |
+| 1970-01-01T00:00:10 | 4   | host2 | idc2       |
+| 1970-01-01T00:00:10 | 5   | host1 | idc3:zone2 |
+| 1970-01-01T00:00:10 | 6   | host2 | idc3       |
+| 1970-01-01T00:00:15 | 1   | host1 | idc1       |
+| 1970-01-01T00:00:15 | 2   | host2 | idc1       |
+| 1970-01-01T00:00:15 | 3   | host1 | idc2:zone1 |
+| 1970-01-01T00:00:15 | 4   | host2 | idc2       |
+| 1970-01-01T00:00:15 | 5   | host1 | idc3:zone2 |
+| 1970-01-01T00:00:15 | 6   | host2 | idc3       |
+| 1970-01-01T00:00:15 | 7   | host1 | idc4:zone3 |
+| 1970-01-01T00:00:15 | 8   | host2 | idc4       |
++---------------------+-----+-------+------------+
+
+-- Empty regex with not existing source label--
+-- SQLNESS SORT_RESULT 3 1
+TQL EVAL(0, 15, '5s') {__name__="test",host="host1"} * label_replace(vector(1), "host", "host2", "host", "");
+
+++
+++
+
+-- SQLNESS SORT_RESULT 3 1
+TQL EVAL(0, 15, '5s') {__name__="test",host="host1"} * label_replace(vector(1), "addr", "host1", "instance", "");
+
```

Comment:
Raise an error now, but Prometheus will ignore it. The binary operator should tolerate this case. cc @waynexia

Reply:
looks like `addr` is from `label_replace` 🤔